tests/Rules.Framework.Tests/RulesEngineTests.cs

```diff
@@ -74,6 +74,49 @@ public async Task AddRuleAsync_GivenEmptyRuleDataSource_AddsRuleSuccesfully()
                 It.Is<EvaluationOptions>(eo => eo == evaluationOptions)), Times.Never());
         }
 
+        [Fact]
+        public async Task DeactivateRuleAsync_GivenEmptyRuleDataSource_DeactivatesRuleSuccessfully()
```

Comment:
@carlosgoias can you please add a unit test for `ActivateRuleAsync(...)`?

Reply:
Done